core/src/com/unciv/logic/map/mapgenerator/MapGenerator.kt

```diff
@@ -19,7 +19,7 @@ import kotlin.random.Random
 class MapGenerator(val ruleset: Ruleset) {
     companion object {
         // temporary instrumentation while tuning/debugging
-        const val consoleOutput = false
+        const val consoleOutput = true
```

Comment:
Do we want this?

Reply:
Maybe? No, in just for testing. Although three months ago the output was permanently on...